lib/workers/miq_worker_types.rb

```diff
@@ -54,6 +54,7 @@
   "ManageIQ::Providers::Vmware::InfraManager::RefreshWorker"                  => %i(manageiq_default),
   "ManageIQ::Providers::Vmware::NetworkManager::RefreshWorker"                => %i(manageiq_default),
   "EmbeddedAnsibleWorker"                                                     => %i(manageiq_default),
+  "AmazonAgentManagerWorker"                                                  => %i(manageiq_default),
```

Comment:
@jrafanie Can you verify the right value here?

Reply:
Looks good to me.  manageiq_default excludes the ui-classic code and dependencies so it should be fine.